staging/src/k8s.io/client-go/tools/events/interfaces.go

```diff
@@ -45,6 +45,11 @@ type EventBroadcaster interface {
 	// NewRecorder returns an EventRecorder that can be used to send events to this EventBroadcaster
 	// with the event source set to the given event source.
 	NewRecorder(scheme *runtime.Scheme, reportingController string) EventRecorder
+
+	// StartEventWatcher enables you to watch for emitted events without usage
+	// of StartRecordingToSink. This lets you also process events in a custom way (e.g. in tests).
+	// NOTE: events received on your eventHandler should be copied before being used
```

Comment:
I actually think it's possible to get rid of id from the interface, but I don't want to block this PR before code freeze.

Can you please add a TODO to figure out if it can be removed?

Reply:
s/id/it/